package.json

```diff
@@ -9,12 +9,14 @@
   },
   "scripts": {
     "build": "webpack --mode production",
-    "start": "npm run build && node ./src/server/bin/server",
+    "start": "npm run db:migrate && npm run build && node ./src/server/bin/server",
```

Comment:
Better not to change The "start" script. Probably no need to do migration here

Reply:
Thank you, I fixed this.